doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -307,21 +307,16 @@ You can schedule actors or tasks to a placement group using
 
 .. note::
 
-  When you use an actor with a placement group, always specify ``num_cpus``.
+  By default, Ray actors use 1 logical CPU for scheduling, and 0 logical CPU for running.
+  This means, by default, actors cannot get scheduled on a zero-cpu node, but an infinite number of them can run on any non-zero cpu node.
+  The default resource requirements for actors was chosen for historical reasons.
+  When scheduling an actor with default resource requirements and a placement group, the placement group has to be created with a bundle containing at least 1 CPU
+  (since the actor requires 1 CPU for scheduling). However, after the actor is created, it doesn't consume any placement group resources.
 
-  When you don't specify (e.g., ``num_cpus=0``), a placement group option is ignored,
-  and the task and actor don't use the reserved resources.
-  
-  Note that by default (with no arguments to ``ray.remote``),
+  To avoid any surprises, always specify resource requirements explicitly for actors. If resources are specified explicitly, they are required for both scheduling and running.
```

Comment:
```suggestion
  To avoid any surprises, always specify resource requirements explicitly for actors. If resources are specified explicitly, they are required both at schedule time and at execution time.
```